java/daikon/dcomp/DCInstrument.java

```diff
@@ -1812,26 +1856,70 @@ void add_exit(MethodGen mg, MethodInfo mi, int method_info_index) {
     }
   }
 
+  /**
+   * Search the interfaces of a class for an implementation of the target method. For each interface
```

Comment:
"For an implementation": might it also be a declaration, without a body?

Reply:
good question.  I assume the target of an invoke instruction but be an implementation?

Reply:
I don't know the answer to this.  Sorry.